index.js

```diff
@@ -347,8 +347,12 @@ const captureWebsite = async (input, options) => {
 			/* eslint-enable no-undef */
 		});
 
-		// Some extra delay to let images load
-		await page.waitForFunction(imagesHaveLoaded, {timeout: 60});
+		try {
+			// Some extra delay to let images load
+			await page.waitForFunction(imagesHaveLoaded, {timeout: timeoutInSeconds});
```

Comment:
This function has been deprecated. Please check out the latest changes in https://github.com/sindresorhus/capture-website/pull/49